pkg/kubernetes/scripts/projects.js

```diff
@@ -94,12 +96,95 @@
         }
     ])
 
+    .factory("projectUtil", [
+        function() {
+            function toName(object) {
```

Comment:
Lets just make this a top level function in this file. If it's not used by anything outside this file.